_sources/canasta/DockerSettings.php

```diff
@@ -36,6 +36,7 @@ function isEnvTrue( $name ): bool {
 ];
 
 const DOCKER_EXTENSIONS = [
+	'SemanticMediaWiki', // keep it at the top to be enabled first, because some Semantic extension don't work in other case.
```

Comment:
sounds interesting, I am curious if this was actually tested? won't extensions dependent on the SMW require `enableSemantic()` to be executed first? due to extension load queue it sounds like order of calling `wfLoadExtension` should not matter much, I might be wrong though